### PR TITLE
Add docs for developer workflow

### PR DESCRIPTION
**Pull request by brenetic**
Document the workflow that is currently used with special focus on the
Testable Editors

### CODE REVIEW

**Review by mattempty — APPROVED** [review]
🚀 

Thanks @brenetic

File: source/documentation/how-we-work.html.md.erb [hunk 1]
```diff
@@ -44,6 +44,36 @@ for how we prefer to work on code.
...
+
+![workflow columns](/images/workflow-columns.png "diagram of the workflow columns")
+
+The purpose of each column are:
+
+**In Progress** - The ticket is currently being worked on.
+
+**Blocked**  - Unable to progress with the work due to some issues. Please add a comment explaining why the ticket is blocked and get help to unblock the ticket.
+
+**In Testable Editor** - This column is applicable only to the tickets related to the Editor app. The developer creates a Testable Editor based off of the code specific to their story. This generates a URL which can be visited for testing by any interested party. Developer should add the Testable Editor URL to the ticket.
```
Comment (njseeto) [inline source/documentation/how-we-work.html.md.erb:59]:
```suggestion
**In Testable Editor** - This column is applicable only to the tickets related to the Editor app. The developer creates a [Testable Editor](https://github.com/ministryofjustice/fb-editor/pull/476) based off of the code specific to their story. This generates a URL which can be visited for testing by any interested party. Developer should add the Testable Editor URL to the ticket.
```

Comment (njseeto) [reply source/documentation/how-we-work.html.md.erb:59]: I've linked to the PR about how to set up a testable editor - might be good for future devs

**Comment by njseeto** [comment]
Looks good - just one small comment

File: source/documentation/how-we-work.html.md.erb (hunk 1 above)
Comment (brenetic) [reply source/documentation/how-we-work.html.md.erb]: 👍  thanks for that. have added a link to the testable editor description in the tech docs themselves

**Review by njseeto — APPROVED** [review]
🚀

**Pull request merged**